src/fh_model.jl

```diff
@@ -18,7 +18,7 @@ function FH_smooth_term()
   pars_FH = [0.5, 0.08, 1.0, 0.8, 0.7]
   prob_FH = DifferentialEquations.ODEProblem(FH_ODE, u0, tspan, pars_FH)
 
-  x0 = [0, 0.2, 1.0, 0, 0]
+  x0 = [0.01, 0.2, 1.0, 0.01, 0.01]
```

Comment:
Maybe this should be an option? Or a different problem?

Reply:
I did this for the same reason as in BPDN, but you're right, there is no problem in using a starting point that is at the edge of the constrained domain. I set back x0 to its previous value !